### DISCUSSION TITLE
Remove Layer Overrides?

### DISCUSSION THREAD

**Discussion by neilberard** [nxt-dev/nxt_editor#195] [Q&A]
Hi there, 
Once I add an override to a referenced layer is there a way to revert that override?
Awesome tool, btw!

**Comment by neilberard** [comment]
Oh, I think I just need to delete the node in the parent layer and that reverts it to the reference.

**Comment by ImLucasBrown** [comment]
Yes you sure can! Right click attributes or nodes and select revert. Deleting a node also works, but revert keeps the node path on the target layer with no data.
_Revert attr_
![revert_attr](https://user-images.githubusercontent.com/54835354/123350240-2f0d4280-d520-11eb-96e5-bab261667e68.gif)
_Revert node_
![revert_node](https://user-images.githubusercontent.com/54835354/123350343-68de4900-d520-11eb-85b7-2a34d69112ed.gif)

**Reply by neilberard** [reply]
Thank you so much!

**Reply by ImLucasBrown** [reply]
No problem!
